benchmarks/zdc_photon/analysis/zdc_photon_plots.py

```diff
@@ -57,16 +57,19 @@ def gauss(x, A,mu, sigma):
     slc=abs(bc-p)<10
     fnc=gauss
     p0=[100, p, 10]
-    coeff, var_matrix = curve_fit(fnc, list(bc[slc]), list(y[slc]), p0=p0,
-                                 sigma=list(np.sqrt(y[slc])+(y[slc]==0)), maxfev=10000)
-    if p==100:
-        xx=np.linspace(p*0.75,p*1.25, 100)
-        plt.plot(xx, fnc(xx,*coeff))
-    pvals.append(p)
-    resvals.append(np.abs(coeff[2])/coeff[1])
-    dresvals.append(np.sqrt(var_matrix[2][2])/coeff[1])
-    scalevals.append(np.abs(coeff[1])/p)
-    dscalevals.append(np.sqrt(var_matrix[2][2])/p)
+    try:
+        coeff, var_matrix = curve_fit(fnc, list(bc[slc]), list(y[slc]), p0=p0,
+                                      sigma=list(np.sqrt(y[slc])+(y[slc]==0)), maxfev=10000)
+        if p==100:
+            xx=np.linspace(p*0.75,p*1.25, 100)
+            plt.plot(xx, fnc(xx,*coeff))
+        pvals.append(p)
+        resvals.append(np.abs(coeff[2])/coeff[1])
+        dresvals.append(np.sqrt(var_matrix[2][2])/coeff[1])
+        scalevals.append(np.abs(coeff[1])/p)
+        dscalevals.append(np.sqrt(var_matrix[2][2])/p)
+    except RuntimeError as e:
+        print(f"fit failed for p={p}", e)
```

Comment:
Would help debugging to plt.savefig here.

Reply:
Not sure if there is an easy way to swap to another figure. For now, left debugging to a printout.